elyra/pipeline/kfp/processor_kfp.py

```diff
@@ -43,7 +46,7 @@
 from traitlets import default
 from traitlets import Unicode
 
-RUN_ID_PLACEHOLDER = "random-placeholder"
+RUN_ID_PLACEHOLDER = '{{workflow.uid}}'
```

Comment:
Relevance of [ELYRA_RUN_NAME](https://elyra.readthedocs.io/en/stable/user_guide/best-practices-file-based-nodes.html#elyra-run-name).

How it compiles during execution:
<img width="1195" alt="Screenshot 2024-04-17 at 4 35 27 PM" src="https://github.com/opendatahub-io/elyra/assets/23582438/0b08dbbf-2616-457b-a471-040605ad7210">

Reply:
thanks :+1: